### PR TITLE
connected to postgres, unflattened data for front end

### PR DESCRIPTION
**Pull request by JasperCDev**
@jayrbarr @timothydadams

### CODE REVIEW

**Review by jayrbarr — APPROVED** [review]
Nice job! Lgtm

File: database/config.js [hunk 1]
```diff
@@ -1,94 +1,28 @@
...
-
-module.exports.deleteGame = (obj, cb) => {
-  Game.deleteOne(obj, (err, result) => {
-    if (err) {
-      cb(err);
-    } else {
-      cb(null, result);
-    }
-  });
+module.exports = {
```
Comment (timothydadams) [inline database/config.js:26]: it looks like you've got some module.exports sprinkled throughout the file, might be easier for future you and others to read if you do it all the same way (put all functions into module.exports, or export them individually), overall looks good though

Comment (JasperCDev) [reply database/config.js:26]: Thank you! that's so weird, those aren't in vscode on my machine. I'll have to look into that.

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
+DROP DATABASE IF EXISTS games;
+
+CREATE DATABASE games;
+
+\c games;
+
+DROP TABLE IF EXISTS gamesTable;
```
Comment (timothydadams) [inline schema.sql:7]: I don't believe its necessary to drop a table for newly created database.  when you source that .sql (or .cql) file, dropping the database on the first line gets rid off all the associated tables too.

File: server/index.js [hunk 3]
```diff
@@ -9,68 +9,91 @@ app.use(parser.json());
...
+// app.delete('/media', (req, res) => {
+//   const game = req.query;
+//   db.deleteGame({ proxyId: game }, (err, result) => {
+//     if (err) {
+//       console.error(err);
+//     } else {
+//       res.send(result);
+//     }
+//   });
+// });
```
Comment (timothydadams) [inline server/index.js:97]: are you intentionally leaving these verbs non-functional?

**Review by timothydadams — APPROVED** [review]
lgtm, left some comments below for your consideration.

File: schema.sql (hunk 2 above)
Comment (JasperCDev) [reply schema.sql:7]: I agree with you. I haven't looked into it but it kept erroring out when I didn't have that.

File: server/index.js (hunk 3 above)
Comment (JasperCDev) [reply server/index.js:97]: for now, yeah. I just haven't implemented them yet. For my front end to work all I need is the GET endpoint.

**Pull request merged**